### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by shivakcpm**
* Implemented movie details component
* Used react hooks wherever possible
* Migrated class components to functional components wherever is possible

### CODE REVIEW

File: src/components/MovieDetails/MovieDetailsComponent.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <div className="header-wrapper">
+      <div className="image-holder"></div>
+      <div className="content-holder">
+        <div className="top-bar">
+          <div>
+            <strong>netflix</strong>
+            <span>roulette</span>
+          </div>
+          <img src="/public/searchIcon.PNG" onClick={props.goToHome.bind(this, null)} />
```
Comment (alionaSipchuk) [inline src/components/MovieDetails/MovieDetailsComponent.js:14]: let's avoid bind in FC!!!!

**Review by alionaSipchuk — CHANGES_REQUESTED** [review]
Looks good) 
fix issues and let's create custom hook where will be filtered your movies by genre

File: src/components/App/App.js [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
+  :  <HeaderComponent allMovies={movies} setAllMovies={setMovies} setQuery={setQuery} />;
+
   return (
     <React.StrictMode>
       <div className="container">
-        <HeaderComponent allMovies={movies} setAllMovies={setMovies} setQuery={setQuery} />
+      { componentToDisplay }
         <Context.Provider value={{ movies:filteredMovies() }}>
-          <ContentHolderComponent allMovies={movies} setAllMovies={setMovies} />
+          <ContentHolderComponent setMovieDetails= {setMovieDetails} allMovies={movies} setAllMovies= {setMovies} />
```
[the remark this answers is not in the document]
Comment (shivakcpm) [reply src/components/App/App.js]: App component has componentholdercomponent , headercomponent, moviedetailscomponent has childrens. so these components can not communicate directly.

File: .eslintrc [hunk 3]
```diff
@@ -207,7 +207,7 @@
       "no-spaced-func": 2,
       "no-trailing-spaces": 2,
       "no-unneeded-ternary": 2,
-      "object-curly-spacing": [2, "always"],
+      // "object-curly-spacing": [2, "always"],
```
Comment (alionaSipchuk) [inline .eslintrc:210]: remove comment

File: prettier.config.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ module.exports = {
   useTabs: false,
   semi: true,
   singleQuote: true,
-  bracketSpacing: true,
+  // bracketSpacing: true,
```
Comment (alionaSipchuk) [inline prettier.config.js:7]: ditto

**Review by alionaSipchuk — APPROVED** [review]
Good job)

**Pull request merged**